danspeech/pretrained_models/streaming_model_GPU.py

```diff
@@ -0,0 +1,17 @@
+import torch
```

Comment:
torch has been disabled, in non streaming models but not here - is that intentional?